packages/server/wsHandler.ts

```diff
@@ -104,8 +108,7 @@ export const wsHandler = makeBehavior<{token?: string}>({
     if (socketCount === 0) {
       handleFirstConnection(user, teamIds).catch(Logger.log)
     }
-    const freshToken = setFreshTokenIfNeeded(extra, user.tms)
-    return {version: __APP_VERSION__, authToken: freshToken}
+    return {version: __APP_VERSION__}
```

Comment:
except for logging in, changing teams, etc., do we ever refresh the cookie?
by removing the fresh token here, it'll cause a lot more folks to get logged out.

in the wsHandler.upgrade, could we refresh the cookie there, before calling res.upgrade?